### DISCUSSION TITLE
How to setup proxy?

### DISCUSSION THREAD

**Discussion by Tutuldot** [sinaptik-ai/pandas-ai#259] [Q&A] [answered]
Hi,

How to configure proxy connection ?

Thanks in advance.

Aze

**Comment by gventuri** [comment]
Hi @Tutuldot, at the moment, unfortunately, it's not possible to configure a proxy connection. There's currently an open issue for that #184!

**Comment by mspronesti** [comment, marked as answer]
@Tutuldot #321 addresses this issue :)
